api-catalog-ui/frontend/package.json

```diff
@@ -86,7 +86,8 @@
         "react-testing-library": "5.3.0",
         "rimraf": "3.0.2",
         "source-map-explorer": "2.3.1",
-        "start-server-and-test": "1.10.8"
+        "start-server-and-test": "1.10.8",
+        "yaml": "^1.10.2"
```

Comment:
Please use the version without ^

Reply:
I am not sure what that means, we installed via `npm install yaml --save-dev`

Reply:
should we omit that manually?